test-app/openapi/v1/openapi.json

```diff
@@ -89,7 +113,14 @@
         "description": "Creates a new blog from provided data",
         "operationId": "createBlog",
         "parameters": [
-
+          {
+            "name": "blog",
```

Comment:
@stevegeek The [issue](https://github.com/rswag/rswag/pull/723#issuecomment-2410490552) we're discussing in thread is also happening here.